src/test/java/org/takes/misc/HrefTest.java

```diff
@@ -72,6 +73,22 @@ void buildsUriWithoutParams() {
         );
     }
 
+    /**
+     * Href can get query parameters.
+     */
+    @Test
+    void getQueryParameters() {
```

Comment:
@baudoliver7 
It would be better to describe behavior in the third person, i.e "extractsParamtetersFromQuery".